mega-service-agentQnA/launch_agents.sh

```
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Script to launch the multi-agent system for AgentQnA
+
+# Define working directory if not already set
+if [ -z "$WORKDIR" ]; then
+  WORKDIR=/Users/jduarte/Documents/GenAIBootcamp/free-genai-bootcamp-2025/mega-service-agentQnA
+fi
+
+# Ensure we're in the right directory
+cd $WORKDIR/GenAIComps
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)